modules/lib_ic/src/ic_low_level.c

```diff
@@ -257,7 +293,8 @@ void ic_adaption_controller(ic_state_t *state, uint8_t vad){
 
     const float_s32_t one = {1, 0};
     const float_s32_t zero = {0, 0};
-    const float_s32_t delta = {1100, -40}; //1100 * 2**-40 = 0.000000001 (from stage_b.py)
+    //const float_s32_t delta = {1100, -40}; //1100 * 2**-40 = 0.000000001 (from stage_b.py)
+    const float_s32_t delta = {352, -45}; //352 * 2**-45 ~ 0.00000000001
```

Comment:
This delta is being used to prevent zero division. In the test case I saw, the denominator was 1.7e-8 while the previous delta was 1e-9. In some cases, it was significant enough to make the fast ratio more than 1 where it shouldn't be so. The test was failing with this delta. I've changed it in both C and python to the less significant one 1e-11, and the test has passed. I'll add a comment to cover that.

Reply:
That makes sense. Since we're changing an IC parameter, do ensure that the full pipeline tests keyword results don't show anything unexpected.